repos/system_upgrade/el7toel8/actors/repositoriesexcluded/libraries/repositoriesexcluded.py

```diff
@@ -0,0 +1,110 @@
+from leapp import reporting
+from leapp.libraries.common.config import get_product_type
+from leapp.libraries.stdlib import api
+from leapp.models import (
+    CustomTargetRepository,
+    RepositoriesExcluded,
+    RepositoriesFacts,
+    RepositoriesMap,
+)
+
+already_reported_skips = []
+
+
+def _is_optional_repo(repo, custom_target_repos):
+    global already_reported_skips  # pylint: disable-msg=global-statement
+    sys_type = get_product_type("source")
+    suffix = "optional-rpms"
+    if sys_type != "ga":
+        suffix = "optional-{}-rpms".format(sys_type)
+    for enabled_repo in custom_target_repos:
+        if enabled_repo.repoid == repo.to_repoid:
+            if repo.to_repoid not in already_reported_skips:
+                already_reported_skips.append(repo.to_repoid)
+                api.current_logger().info(
+                    "Skipping %s repository from exclusion due to directly "
+                    "specified in --enablerepo flag"
```

Comment:
```suggestion
                    "Skipping %s repository from exclusion as it is directly "
                    "specified in the --enablerepo option"
```

Reply:
After refactoring this message no longer needed